plugins/services/src/js/components/modals/NewCreateServiceModalForm.js

```diff
@@ -90,8 +111,13 @@ class NewCreateServiceModalForm extends Component {
     this.props.onConvertToPod(this.getAppConfig());
   }
 
-  componentDidUpdate() {
-    this.props.onChange(new this.props.service.constructor(this.state.appConfig));
+  componentDidUpdate(prevProps, prevState) {
+    const {editingFieldPath} = this.state;
```

Comment:
🤔  you don't destruct `appConfig` that is being used twice but you do `editingFieldPath` that is being used once